ballerina/tests/52_unary_ldap_auth_client.bal

```diff
@@ -14,125 +14,140 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/jballerina.java;
 import ballerina/test;
 
 @test:Config {
-    enable: false,
+    enable: true,
```

Comment:
Do we need to explicitly mention the `enable: true` here?

Reply:
This has been a practice in gRPC. We'll group those tests later.